.github/workflows/ci.yml

```
@@ -0,0 +1,58 @@
+    runs-on: ubuntu-latest
+    needs: [check]
+    steps:
+      - uses: actions/checkout@v7
+
+      - name: Build
+        run: |
+          make -C web ci
+
+      - name: Run migrations
+        run: make -C web migrate
+        background: true
```

actionlint (1.7.12)

[error] unexpected key "background" for step to run shell command. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "working-directory"

(syntax-check)